config/locales/en.yml

```diff
@@ -1982,6 +1982,7 @@ en:
   highlights_and_stats_from_year_on_site: "Highlights and stats from %{year} on %{site}"
   history: History
   home: Home
+  hours_adjusted: "Hours Adjusted:"
```

Comment:
This could use a context comment, e.g. "Label showing hours adjusted in the time shifter in the Uploader". Same with `offset_time`